Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM gradle:jdk8
+
+COPY ./kelpie.tar .
+
+RUN tar xf kelpie.tar && rm -f kelpie.tar
+
+RUN mv kelpie/bin/* /usr/local/bin/ && \
+    mv kelpie/lib/* /usr/local/lib/ && \
+    rm -rf kelpie
+
+RUN git clone https://github.com/scalar-labs/kelpie-test.git kelpie-test && \
```

Comment:
I think creating `kelpie-test` (client-test) container instead of `kelpie` container with `kelpi-test`.
So I think it's better to have Dockerfile in `kelpie-test/client-test`  so that the dependency to kelpie is resolved by the build.gradle and you don't have to either use `gradle:jdk8` nor do `gradle shadowJar` any more here.

Reply:
fixed in https://github.com/scalar-labs/kelpie/pull/17/commits/35c79c3928d2eb5b54a3ca23138391c5de76b551

Reply:
Sorry, that's not what I mean. Let's chat later.